Lib/test/_test_embed_set_config.py

```diff
@@ -211,6 +212,14 @@ def test_flags(self):
         self.set_config(use_hash_seed=1, hash_seed=123)
         self.assertEqual(sys.flags.hash_randomization, 1)
 
+        if support.Py_GIL_DISABLED:
+            self.set_config(enable_gil=-1)
+            self.assertEqual(sys.flags.gil, None)
+            self.set_config(enable_gil=0)
+            self.assertEqual(sys.flags.gil, 0)
+        self.set_config(enable_gil=1)
```

Comment:
FWIW, as a reader this implies to me that `PyConfig.enable_gil` is available even on builds without `--disable-gil`.  Seeing that it isn't available makes me wonder why this call isn't failing.  Is `self.set_config()` tolerant of unrecognized config fields?